.github/workflows/run-tests.yml

```
@@ -7,22 +7,23 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11-dev"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.14-dev"]
```

Python 3.9 is now EOL, and 3.14 is GA.
```suggestion
        python-version: ["3.10", "3.11", "3.12", "3.13", "3.14"]
```

---

Could also update the classifiers in setup.py, but see also https://github.com/jmespath/jmespath.py/issues/328.